src/app/component/core/search-popup/search-not-found/search-not-found.component.ts

```diff
@@ -0,0 +1,20 @@
+import {Component, EventEmitter, OnInit, Output} from '@angular/core';
+
+@Component({
+  selector: 'app-search-not-found',
+  templateUrl: './search-not-found.component.html',
+  styleUrls: ['./search-not-found.component.scss']
+})
+export class SearchNotFoundComponent implements OnInit {
+  @Output() closeFromChild = new EventEmitter();
+
+  private emitClose(): void {
+    this.closeFromChild.emit('1');
```

Comment:
Please, don't use 'magic numbers'. You can replace it with the boolean value

Reply:
fixed